src/main/java/org/folio/circulation/domain/representations/AccountPaymentStatus.java

```diff
@@ -2,6 +2,7 @@
 
 public enum AccountPaymentStatus {
   CANCELLED_ITEM_RETURNED("Cancelled item returned"),
+  CANCELLED_ITEM_RENEWED("Cancelled item renewed"),
```

Comment:
Is `cancelled` synonymous with `lost`?

Reply:
No, i think what actually meant here is `Cancelled, item renewed`. But requirement was without the comma.